GPUVerifyVCGen/IRaceInstrumenter.cs

```diff
@@ -21,7 +21,7 @@ public interface IRaceInstrumenter
 
         void AddRaceCheckingDeclarations();
 
-        BigBlock MakeResetReadWriteSetStatements(Variable v, Expr resetCondition);
+        BigBlock MakeResetReadWriteSetStatements(Variable v, Expr resetCondition, bool gridBarrier = false);
```

Comment:
Do we need the default value for `gridBarrier`?

Reply:
I assigned the default value so that we don't have to modify all the places where this is currently being called. I can remove the default value if you prefer.

Reply:
I think it's only called in one place, and that place you updated. So, I think it can be removed without having to touch any other code (but let me know if I'm wrong).